### ISSUE TITLE
Back button is too hard to acquire with mouse

### ISSUE THREAD

**Issue by potch** [bwinton/VerticalTabs#142]
Would prefer the tab side-rail begin *below* toolbars, and let the back button keep its Fitts-law-satisfying location.

**Comment by bwinton** [comment]
I'ld definitely like to see this as an experiment, since it would also solve Michael Verdi's problem with new tabs and typing urls…  @ericawright, can you try to quickly whip this up when you get a chance?

**Comment by kodabb** [comment]
i agree, either below or in the same position with just movable items

**Comment by wresuolc** [comment]
This could be a good candidate for the feature-flag a/b testing ability in test pilot. /cc @lmorchard

**Comment by paulkoan** [comment]
This is a routine issue for me.  
1. Go to click back button
2. Overshoot by 15px triggering tabcenter open animation
3. Come back > 15px back to where the Back button *was*
4. Click intending to hit back, but instead hit "New tab"
5. Close new tab
6. Hit "Back" more carefully

**Comment by heftig** [comment]
Putting the tab sidebar on the right instead of the left could also be an option to avoid moving the back button.

BTW, what happens if the UI direction is right-to-left?

**Comment by paulkoan** [comment]
Another option would be to have a longer "open" delay when you mouse into the "New tab / search" area - longer than when choosing a tab.  So if you overshoot the back button, you have a chance to get back to it before the tab center opens up.

My workaround is to have another icon in between the back button and the tab center sidebar.

**Comment by 0xdec** [comment]
Any news on this? Is it on the roadmap or has it been abandoned?

**Comment by bwinton** [comment]
No-one has stepped up to work on it recently.

**Comment by mccoys** [comment]
This issue is particularly annoying when the "auto-shrink" option is activated because the vertical tabs bar keeps hiding the navigation bar.

I would like to concur to the earlier suggestion that the "new tab" icon be below the navigation bar. Not on its left.

In addition, when one particular tab is hovered, do not expand the whole bar, but only that one tab. Even though that does not relate specifically to the "back button" problem, it would help the overall accessibility of the buttons on the top-left corner.
